tests/Microsoft.Bot.Builder.TestBot.WebApi/Dialogs/BookingDialog.cs

```diff
@@ -17,19 +17,26 @@ public BookingDialog()
             AddDialog(new TextPrompt(nameof(TextPrompt)));
             AddDialog(new ConfirmPrompt(nameof(ConfirmPrompt)));
             AddDialog(new DateResolverDialog());
-            AddDialog(new WaterfallDialog(nameof(WaterfallDialog), new WaterfallStep[]
+            var steps = new WaterfallStep[]
             {
                 DestinationStepAsync,
                 OriginStepAsync,
                 TravelDateStepAsync,
                 ConfirmStepAsync,
                 FinalStepAsync,
-            }));
+            };
+            AddDialog(new WaterfallDialog(nameof(WaterfallDialog), steps));
```

Comment:
Why?
Can we just keep this consistent with the original sample?
Specifically, sure if you want to make this change for some reason can you _also_ make it in the sample.